styles/components/_autocomplete.scss

```diff
@@ -0,0 +1,40 @@
+// ==================================================================
+// Autocomplete
+// ==================================================================
+
+////
+/// @group autocomplete
+////
+.sprk-c-Autocomplete__results {
+  list-style-type: $sprk-autocomplete-results-list-style-type;
+  background-color: $sprk-white;
+  border: $sprk-autocomplete-results-border;
+  border-radius: $sprk-autocomplete-results-border-radius;
+  box-shadow: $sprk-autocomplete-results-shadow;
+  max-width: $sprk-autocomplete-results-max-width;
+  max-height: $sprk-autocomplete-results-max-height;
+  position: $sprk-autocomplete-results-position;
+  z-index: $sprk-autocomplete-results-z-index;
+  overflow-y: $sprk-autocomplete-results-overflow-y;
+  -webkit-overflow-scrolling: touch; // iOS momentum scrolling
+}
+
+/// Apply highlighting style to the element.
+.sprk-c-Autocomplete__results--active {
+  background-color: $sprk-autocomplete-results-active-background-color;
+}
+
+.sprk-c-Autocomplete__result {
+  padding: $sprk-autocomplete-results-items-padding;
```

Comment:
These var names should match the classnames. $sprk-autocomplete-result-padding

Reply:
Might have been a typo since you updated the classname to be something different and forgot to update the vars.